backend/business-partner-agent/src/main/java/org/hyperledger/bpa/controller/PartnerController.java

```diff
@@ -216,6 +223,30 @@
         return HttpResponse.ok();
     }
 
+    /**
+     * Aries: Get Proofs requested from a partner
+     *
+     * @param id {@link UUID} the partner id
+     * @return list of pending proof requests from partners
+     */
+    @Get("/{id}/proof-requests")
+    public HttpResponse<String> fetchProofRequests(
+            @PathVariable String id) {
+        List<PresentationExchangeRecord> results = proofM.getProofRequests(Optional.of(UUID.fromString(id)), null)
+                .get();
+
+        String jsonString = "";
+        try {
+            //TODO none of this.... change/use class that is serializable by jackson
+            Gson gson = new GsonBuilder().setPrettyPrinting().create();
+            jsonString = gson.toJson(results);
+
+        } catch (Exception e) {
+        }
+
+        return HttpResponse.ok(jsonString);
```

Comment:
The class the aca-py wrapper is using for the aca-py response is not serialisable by the `jackson` library that micronaut is configured to use. I'm ashamed of this code, but i'm not sure how you see this working. 

There is currently no BPA level record for a pending presentation-request and i'm not sure if there should be. 

Should i made a model just to convert into that we can control? that sounds possible but the jsonview i found for this library need the fields to be bound to the views which doesn't really work given the models are coming from the wrapper and we want to define the view in the layer above. 

`https://www.baeldung.com/jackson-json-view-annotation`

Or should we just modify the class `https://github.com/boschresearch/aries-acapy-clients/blob/main/java/src/main/java/org/hyperledger/aries/api/present_proof/PresentationExchangeRecord.java` to use a jackson.JavaObject (assuming there is an equivalent)?

Or should we build the proper model for the `presentation` attribute so that we don't have to use any abstract `jsonObject`.

@etschelp Looking for some design help as i'm not sure which direction you want to go.